### PR TITLE
eip27-edits: come corrections and todos

### PR DESCRIPTION
**Pull request by aslesarenko**

### CODE REVIEW

File: eip-0027.md [hunk 1]
```diff
@@ -84,12 +84,6 @@ Contracts
 
     val correctNftId = EQ(firstTokenId, ByteArrayConstant(reemissionNftId))
     
-    // output of the reemission contract
```
Comment (kushti) [inline eip-0027.md:87]: why removed? reemissionOut and minerOut used below: 

```
    // miner's output must have a script which is the time-locking reward for the miner's pubkey
    // box height must be the same as block height
    val correctMinerOutput = AND(
      EQ(ExtractScriptBytes(minerOut), expectedMinerOutScriptBytesVal(monetarySettings.minerRewardDelay, MinerPubkey)),
      EQ(Height, boxCreationHeight(minerOut))
    )
    
    // reemission output's height must be the same as block height
    val heightCorrect = EQ(boxCreationHeight(reemissionOut), Height)
```

File: eip-0027.md [hunk 2]
```diff
@@ -113,11 +107,15 @@ Contracts
     val correctCoinsIssued = EQ(reemissionRewardPerBlock, Minus(ExtractAmount(Self), ExtractAmount(reemissionOut)))
     
     // when reemission contract box got merged with other boxes
-    val sponsored = AND(
+    // TODO: describe why this condition can replace 5 others (heightCorrect, etc)
```
Comment (kushti) [inline eip-0027.md:110]: it is not replacing them, rather, an alternative branch. So re-emission box is whether getting additional funding via `sponsored` path, or spending funds according to rules (heightCorrect etc)

**Review by kushti — CHANGES_REQUESTED** [review]

File: eip-0027.md (hunk 1 above)
Comment (aslesarenko) [reply eip-0027.md:87]: But they are declare twice, see above.

File: eip-0027.md (hunk 2 above)
Comment (aslesarenko) [reply eip-0027.md:110]: I don't understand what `sponsored` means, what is the intuition about this. Better to put it into the contract. 
Without understanding the intended logic, it is hard to review and estimate correctness of the contract.

Comment (kushti) [reply eip-0027.md:110]: Contract description in this EIP have this: 
```
Also, this contract allows for merging with other boxes; the merging transaction must have only two outputs; the first output is for the re-emission contract, the second one is to pay a mining fee (its value can be 0.01 ERG at most)
```

Comment (aslesarenko) [reply eip-0027.md:110]: Ok, I've read that. Why not also hint about what are the inputs of the merging transaction?

Comment (kushti) [reply eip-0027.md:110]: Well, there are no limits there, so re-emission contract box can be merged with any inputs allowing for outputs specified.

Comment (aslesarenko) [reply eip-0027.md:110]: Ok, the comment above inputs would be helpful in the code to decipher the contract.

**Pull request closed without merging**